docs/developer/architecture/core/uisettings-service.asciidoc

```diff
@@ -26,6 +26,16 @@ Configuration through the Advanced Settings UI is restricted to users authorised
 === Configuration with UI settings overrides
 experimental[] When a setting is configured as an override in kibana.yml, it will override any other value stored in the config saved object. If an override is misconfigured, it will fail config validation and prevent Kibana from starting up. The override applies to Kibana as a whole for all spaces and users and the option will be disabled in the Advanced Settings page. We refer to these as "global" overrides.
 
+Use the top-level `uiSettings` key for this, for example:
+
+[source,yaml]
+----
+# Fewer marketing messages
```

Comment:
I'd suggest using the wording like it's used in Advanced settings, slightly adapted

```suggestion
# Less highlighting new features messages
```

Reply:
Or maybe
```suggestion
# Less feature announcement messages
```